README.md

```diff
@@ -2,7 +2,7 @@
 
 [中文阅读](./README_zh.md)
 
-Fine-tune Cog family of video models for custom video generation under 24GB of GPU memory ⚡️📼
+Fine-tune Cog family of video models for custom video generation under 24GB of GPU memory, support multi resolutions ⚡️📼
```

Comment:
I think for headline we can just leave it to what it was before. There can be a separate section for supported capabilities once we add a few more things

```suggestion
Fine-tune Cog family of video models for custom video generation under 24GB of GPU memory ⚡️📼
```

Reply:
use your's is correct

Reply:
Agree with the suggestion.